Assets/MixedRealityToolkit-Tests/MixedRealityManagerTests.cs

```diff
@@ -0,0 +1,403 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Internal.Managers;
+using NUnit.Framework;
+using UnityEngine;
+using UnityEngine.TestTools;
+
+namespace Microsoft.MixedReality.Toolkit.Tests
+{
+    public class MixedRealityManagerTests
+    {
+        public void CreateMRManager()
+        {
+            // Create The MR Manager
+            GameObject go = GameObject.Find("Mixed Reality Manager");
+            if (!go)
+            {
+                go = new GameObject("Mixed Reality Manager");
+            }
+            go.AddComponent<MixedRealityManager>();
+        }
+
+        [Test]
+        public void Test01_CreateMixedRealityManager()
```

Comment:
Should we put `Test<number>_` prefix on these?

Reply:
Yes, because Unity CANNOT sort Tests :D